pyuvdata/tests/test_uvbeam.py

```diff
@@ -1286,30 +1286,19 @@ def test_healpix():
     nt.assert_true(np.allclose(YY_area / XX_area, np.ones(numfreqs)))
     # nt.assert_almost_equal(YY_area / XX_area, 1.0, places=5)
 
-    # check backwards compatability with pstokes nomenclature and int polnum
-    I_area = healpix_norm.get_beam_area('I')
-    pI_area = healpix_norm.get_beam_area('pI')
-    area1 = healpix_norm.get_beam_area(1)
-    nt.assert_true(np.allclose(I_area, pI_area))
-    nt.assert_true(np.allclose(I_area, area1))
-
     # Check that if pseudo-Stokes I (pI) is in the beam polarization_array, it just uses it
     healpix_norm.polarization_array = [1, 2]
-    nt.assert_almost_equal(np.sum(healpix_norm.get_beam_area()), 2. * numfreqs * npix * dOmega)
-    nt.assert_almost_equal(np.sum(healpix_norm.get_beam_sq_area()), 4. * numfreqs * npix * dOmega)
+    # nt.assert_almost_equal(np.sum(healpix_norm.get_beam_area()), 2. * numfreqs * npix * dOmega)
```

Comment:
If these tests aren't needed, just delete them rather than commenting them out. Git will preserve the memory of them if we need.

Reply:
Deleted!